spec/factories/purchase_forms.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :purchase_form do
+    token {"tok_abcdefghijk00000000000000000"}
+    item_id { 1 }
+    user_id { 1 }
```

Comment:
```suggestion
    item_id { 1 }
    user_id { 1 }
```
商品とユーザーのIDを固定で1とするのはテストを行う上で望ましくないため削除しておきましょう。